app/celery/provider_tasks.py

```diff
@@ -81,17 +81,16 @@ def deliver_email(self, notification_id):
         _check_and_queue_callback_task(notification)
     except MalwareDetectedException:
         _check_and_queue_callback_task(notification)
-    except Exception as e:
-        if isinstance(e, MalwareScanInProgressException) and self.request.retries <= SCAN_MAX_BACKOFF_RETRIES:
-            countdown = SCAN_RETRY_BACKOFF * (self.request.retries + 1)  # do we need to add 1 here?
+    except MalwareScanInProgressException:
+        if self.request.retries <= SCAN_MAX_BACKOFF_RETRIES:
+            countdown = SCAN_RETRY_BACKOFF * (self.request.retries + 1)
         else:
             countdown = None
         try:
-            current_app.logger.warning(f"The exception is {repr(e)}")
```

Comment:
Did we add this log warning in a specific context? This is not informative of the context. Nonetheless, I added the exception to the `parameter` while logging the exception down below.

Reply:
This was probably debugging code that slipped through